cpp/include/Ice/Communicator.h

```diff
@@ -199,129 +182,121 @@ namespace Ice
         /// @see Connection::setAdapter
         void setDefaultObjectAdapter(ObjectAdapterPtr adapter);
 
-        /// Get the implicit context associated with this communicator.
-        /// @return The implicit context associated with this communicator; returns null when the property
-        /// Ice.ImplicitContext is not set or is set to None.
+        /// Gets the implicit context associated with this communicator.
+        /// @return The implicit context associated with this communicator; returns `nullptr` when the property
+        /// `Ice.ImplicitContext` is not set or is set to `None`.
         [[nodiscard]] ImplicitContextPtr getImplicitContext() const noexcept;
 
-        /// Get the properties for this communicator.
+        /// Gets the properties of this communicator.
         /// @return This communicator's properties.
-        /// @see Properties
         [[nodiscard]] PropertiesPtr getProperties() const noexcept;
 
-        /// Get the logger for this communicator.
+        /// Gets the logger of this communicator.
         /// @return This communicator's logger.
-        /// @see Logger
         [[nodiscard]] LoggerPtr getLogger() const noexcept;
 
-        /// Get the observer resolver object for this communicator.
-        /// @return This communicator's observer resolver object.
+        /// Gets the observer object of this communicator.
+        /// @return This communicator's observer object.
         [[nodiscard]] Instrumentation::CommunicatorObserverPtr getObserver() const noexcept;
 
-        /// Get the default router for this communicator.
-        /// @return The default router for this communicator.
-        /// @throws CommunicatorDestroyedException if the communicator has been destroyed.
+        /// Gets the default router of this communicator.
+        /// @return The default router of this communicator.
+        /// @throws CommunicatorDestroyedException Thrown when the communicator is destroyed.
```

Comment:
`has been` feels more right to me. Otherwise it reads like
"when you destroy the communicator, we will throw this exception" (the present tense)
when what we want is the "perfect present passive tense":
> X will happen now if Y happened in the past.

```suggestion
        /// @throws CommunicatorDestroyedException Thrown when the communicator has been destroyed.
```